pkg/operator/http.go

```diff
@@ -27,10 +27,12 @@ import (
 	listers "github.com/operator-framework/operator-metering/pkg/generated/listers/metering/v1alpha1"
 	"github.com/operator-framework/operator-metering/pkg/operator/prestostore"
 	"github.com/operator-framework/operator-metering/pkg/presto"
+	"github.com/operator-framework/operator-metering/pkg/util/chiprometheus"
 	"github.com/operator-framework/operator-metering/pkg/util/orderedmap"
 )
 
 var ErrReportIsRunning = errors.New("the report is still running")
+var promRouter = chiprometheus.NewMiddleware("addmiddleware")
```

Comment:
`"addmiddleware"` is the `service` label in the metric so this isn't very accurate. We should probably use `"reporting-operator"` to indicate the service this metric comes from is `reporting-operator`.

Reply:
Also this variable name is very inaccurate. It should probably be `prometheusMiddleware` instead of `promRouter`

Reply:
fixed in cd951fd4a39c7435744a0dc432904f20a20ad76d